Gemfile.jruby-1.9.lock.release

```diff
@@ -4,13 +4,15 @@ PATH
     logstash-core (5.5.0-java)
       chronic_duration (= 0.10.6)
       clamp (~> 0.6.5)
-      concurrent-ruby (= 1.0.0)
+      concurrent-ruby (~> 1.0, >= 1.0.5)
+      elasticsearch (~> 5.0, >= 5.0.4)
```

Comment:
I was curious about elasticsearch dependency. It came in as part of 351069f9 (#7345).